tests/python/contrib/test_vitis_ai_runtime.py

```diff
@@ -0,0 +1,104 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+# pylint: disable=no-else-return, unidiomatic-typecheck, invalid-name, W0611
+
+""" Vitis-AI runtime test """
+
+import sys
+import numpy as np
+
+import pyxir
+import pyxir.contrib.target.DPUCADX8G
+
+import tvm
+import tvm.relay.testing
+from tvm import relay
+from tvm import runtime
+from tvm.relay import transform
+from tvm.contrib import util
+from tvm.relay.backend import compile_engine
+from tvm.relay.build_module import bind_params_by_name
+from tvm.relay.op.contrib.vitis_ai import annotation
+from tvm.contrib.target import vitis_ai
+
+
+
+def check_result(mod, map_inputs, out_shape, result, tol=1e-5, target="llvm",
+                 ctx=tvm.cpu(), params=None):
+    """Check the result between reference and generated output with vitis-ai byoc flow"""
+    if sys.platform == "win32":
+        print("Skip test on Windows for now")
+        return
```

Comment:
Put this check in `test_extern_vai_resnet18` directly so that we won't waste time running that test on Windows.

Reply:
@mbaret @comaniac  I've updated per your comments. Could you take another look?